.envrc

```diff
@@ -25,11 +25,12 @@ export FISSILE_RELEASE="${FISSILE_RELEASE#,}"
 export FISSILE_ROLE_MANIFEST="${PWD}/role-manifest.yml"
 export FISSILE_LIGHT_OPINIONS="${PWD}/opinions.yml"
 export FISSILE_DARK_OPINIONS="${PWD}/dark-opinions.yml"
-export FISSILE_WORK_DIR="${PWD}/work-dir"
+export FISSILE_WORK_DIR="$HOME/.fissile/uaa-work-dir"
 export FISSILE_REPOSITORY="uaa"
+export FISSILE_CACHE_DIR="$HOME/.bosh/cache"
```

Comment:
Oh, right, when it becomes a submodule lots of this can go away (because `source_up`).